signcrypt_seal.go

```diff
@@ -69,6 +69,7 @@ func (sss *signcryptSealStream) signcryptBytes(b []byte) error {
 
 	nonce := nonceForChunkSigncryption(sss.numBlocks)
 
+	// TODO: Encode payload chunk.
```

Comment:
Yep, I'd say throw a single byte into the signature input, wherever you like. I think all the inputs are constant length, since the plaintext chunk is hashed by itself first.